web/package.json

```diff
@@ -16,6 +16,7 @@
     "react-qr-code": "^2.0.2",
     "react-qr-reader": "^2.2.1",
     "styled-components": "^5.3.1",
+    "universal-cookie": "^4.0.4",
```

Comment:
I'm not sure, is it possible to use only one cookie lib? js-cookie or universal-cookie.
https://www.libtrends.info/npm-compare/js-cookie-vs-react-cookie-vs-react-cookies-vs-redux-cookies-vs-universal-cookie